### ISSUE TITLE
Allow to resize views with keybindings only

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#22645]
Provide commands to resize the active view using keyboard only.

**Comment by cleidigh** [comment]
perfect, thanks

if I can get my debug working across repositories I can get onto this...

**Comment by bpasero** [comment]
@cleidigh what is the issue you are seeing?

**Comment by cleidigh** [comment]
see #22642 
I've been working on it/confused as hell for the last three days
just concluded fresh check out/direct clone does not dbg but no error output
the executable just launches

**Comment by cleidigh** [comment]
@bpasero 
you can sign me up for this officially...

**Comment by bpasero** [comment]
@cleidigh looks like the workaround for you is to debug from one workspace to work on this.

**Comment by cleidigh** [comment]
@bpasero 
and now back to our regularly scheduled programming...

Sorry, the debug problem and SCM/keyboard issues sidetracked me tremendously
but I learned a lot - that's a good thing the nature of the beast.

I got a cleaned up branch of my first resize implementation. I think it's
ready for a pull. I want to make sure I get feedback before I go too far
given this is my first pull. Note: it's also my first git project ,
it wasn't around when I was doing my work before (shows my age ;-)

This prototype works on the sidebar, the panel and one editor.
I'll comment more in the pull
